packages/components/Button/src/useButton.ts

```diff
@@ -34,11 +32,3 @@ export const useButton = (props: ButtonProps): ButtonState => {
     state: pressable.state,
   };
 };
-
-const getAccessibilityState = memoize(getAccessibilityStateWorker);
-function getAccessibilityStateWorker(disabled: boolean, accessibilityState?: AccessibilityState) {
-  if (accessibilityState) {
-    return { disabled: disabled, ...accessibilityState };
-  }
-  return { disabled: disabled };
```

Comment:
Pressable does this for us (minus the memoization :( )

https://github.com/facebook/react-native/blob/b55518c4bf22ea6f409b58151d9c84fa161d6e02/Libraries/Components/Pressable/Pressable.js#L206